gamma/2.1/Dockerfile

```diff
@@ -0,0 +1,107 @@
+# base image
+FROM ubuntu:bionic as app
+
+# for easy upgrade later. ARG variables only persist during image build time
+ARG GAMMA_VER="2.1"
+
+# metadata
+LABEL base.image="ubuntu:bionic"
+LABEL dockerfile.version="2"
+LABEL software="centroid"

```

Comment:
Please change to: `LABEL software="GAMMA"`

and the above line 9 to `LABEL dockerfile.version="1"` since this is a new dockerfile for v2.1 of GAMMA